### PR TITLE
Update DB schema files

### LINKED ISSUE #421: Update DB schema files

### PR DESCRIPTION
**Pull request by jvalkeal**
- Due to change in skipper package metadata table
- Fixes #421
- Resolves spring-cloud/spring-cloud-dataflow#1913

### CODE REVIEW

File: spring-cloud-skipper-server-core/src/main/resources/db-schema/schema-db2.sql [hunk 1]
```diff
@@ -176,7 +177,6 @@
 
     create table skipper_status (
         id bigint not null,
-        object_version bigint,
```
Comment (ilayaperumalg) [inline spring-cloud-skipper-server-core/src/main/resources/db-schema/schema-db2.sql]: I guess `object_version` needs to be removed in some other places where they are still being used?

Comment (jvalkeal) [reply spring-cloud-skipper-server-core/src/main/resources/db-schema/schema-db2.sql]: This is due to changed super class in commit https://github.com/spring-cloud/spring-cloud-skipper/commit/53b83cc0756d10517caca4fadd24807608d71003

**Comment by ilayaperumalg** [comment]
LGTM, merging.

**Comment by ilayaperumalg** [comment]
Rebased and merged as 0d8d7bd

**Pull request closed without merging**